Update release templates (#8108)
owncloud/client: .github/patch_release_template.md

* [ ] QA: Add a link to RC1 to owncloud.com
* [ ] QA: Inform [stakeholders](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunication-InformStakeholdersAfterSuccessfulSprintBuild)

### QA

#### DEV QA
* [ ] DEV: [Smoke test](https://confluence.owncloud.com/display/OG/Manual+Tests#ManualTests-DEVSmokeTest)
* [ ] DEV: Run [automated tests](https://confluence.owncloud.com/display/OG/Automated+Tests)
#### BTR QA
* [ ] QA: Inform subscribers of the beta program about a new version for testing
* [ ] QA: [Antivirus scan](https://confluence.owncloud.com/display/OG/Virus+Scanning)
* [ ] QA: Changelog testing
* [ ] QA: [Regression test](https://confluence.owncloud.com/display/OG/Manual+Tests#ManualTests-RegressionTest)

### After QA Approval

* [ ] DEV: Remove git suffix in `MIRALL_VERSION_SUFFIX` in [`VERSION.cmake`](https://confluence.owncloud.com/display/OG/Branching+Off#BranchingOff-VERSION.cmake)
* [ ] DEV: On sprint branch create a **signed** [tag](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-clientrepository) v2.6.1

> * [ ] DEV: Delete the sprint branch

Reviewer: ```suggestion
```

* [ ] DEV: Create the same tag on branch 2.6 of [_client-plugin-vfs-win_](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-client-plugin-vfs-win)
* [ ] DEV: Create the same tag on branch 2.6 of [_wip-msi_](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-wip-msi)
* [ ] DEV: Create same tag for [Windows toolchain](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-Windowstoolchain)
* [ ] DEV: Create same tag (actually a symlink) for [macOS toolchain](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-macOStoolchain)

### Final Rebuild

* [ ] QA: Change the date in `ChangeLog` to the date of creating the tag 
* [ ] QA: Bump `MIRALL_VERSION_PATCH` in [`VERSION.cmake`](https://confluence.owncloud.com/display/OG/Branching+Off#BranchingOff-PatchRelease)
* [ ] QA: Adjust [Linux Templates](https://confluence.owncloud.com/display/OG/Branching+Off#BranchingOff-Linuxtemplates) to support the next patch release version (e.g. 2.6.2)

> * [ ] QA: Run [client-trigger](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-Finalbuild) 

Reviewer: ```suggestion
* [ ] QA: Trigger the [build job](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-Finalbuild) 
```

* [ ] QA: Check if [GitHub download links](https://github.com/owncloud/client/releases) point to correct location
* [ ] QA: Publish the release in GitHub
* [ ] QA: Check [minimum.supported.desktop.version](https://github.com/owncloud/core/blob/master/config/config.sample.php#L1367) on the server

### Communicate the Availability

* [ ] QA: Announce on [central](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunication-Central) listed in [Category News, desktop tag](https://central.owncloud.org/tags/c/news/desktop)
* [ ] QA: Inform [stakeholders](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunication-InformStakeholdersaboutFinal)
* [ ] QA: Inform [packagers](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunication-Packagers) - ping @dragotin (openSUSE)

### Final Infrastructure Check

* [ ] QA/DEV: Update [stable channel](https://confluence.owncloud.com/display/OG/Online+Updater%2C+Crash+reporter%2C+Transifex#OnlineUpdater,Crashreporter,Transifex-UpdatetheUpdater) in the owncloud hosted auto updater

> * [ ] QA/DEV: Ensure that the [client patch release template](https://github.com/owncloud/client/blob/master/.github/patch_release_template.md) is up to date

Reviewer: ```suggestion
* [ ] QA/DEV: Ensure that the [client patch release template](https://github.com/owncloud/client/blob/master/.github/patch_release_template.md) is up to date and in sync with other client templates
```
